NebulaModel/MultiplayerOptions.cs

```diff
@@ -16,6 +16,10 @@ public class MultiplayerOptions : ICloneable
         [UIRange(1, ushort.MaxValue)]
         public ushort HostPort { get; set; } = 8469;
 
+        [DisplayName("Enable UPnp/Pmp Support")]
+        [Description("If enabled, attempt to automatically create a port mapping using UPnp/Pmp (only works if your router has this feature and it is enabled)")]
+        public bool EnableUPnpOrPmpSupport { get; set; } = true;
```

Comment:
The default option should probably be set to false. Let the user enable this option by themself.

Reply:
I changed it to false by default